src/common/Gateway.sol

```diff
@@ -449,3 +444,20 @@ contract Gateway is Auth, IGateway, Recoverable {
         return inboundBatch[centrifugeId][batchHash].votes;
     }
 }
+
+library MessageProofLib {
```

Comment:
Given it's only used within the same file, I would actually move this to internal methods in the contract, rather than a library.

Reply:
Will do